modules/ROOT/pages/editor-premium-upgrade-promotion.adoc

```diff
@@ -0,0 +1,26 @@
+= Premium upgrade promotion
+:navtitle: {productname} Premium upgrade promotion
+:description: Editor options related to turning the Premium promotion display off
+:keywords:
```

Comment:
Should there be any keywords here? E.g "Upgrade" "button" "promotion" "6.2". I don't even know if these keywords are used anywhere outside the repo but could be useful anyway in case someone needs to find this page in the repo.

Reply:
Good idea. Done.

Reply:
@MitchCTiny just to let you know, these are actually used for SEO to some extent. They are rendered in the `<head>` like so:
```
<meta name="keywords" content="upgrade, promotion, premium, button">
```